### ISSUE TITLE
Fire Tick goes through PVP

### ISSUE THREAD

**Issue by Zunwolf1896** [BlossomMods/BlossomPVP#3]
Hello, have recently noticed that players with PVP off getting hit by something that gives Fire damage over ticks goes through the PVP. The players will both get the "pvp is disabled" message, and wont take the initial damage, but the fire tick damage that comes after will still be taken.

**Comment by CodedSakura** [comment]
am i understanding this correctly? if both players A and B have PVP off, and A hits B with a Fire Aspect sword, both players see a message that the PVP is disabled, but player B still gets set on fire and takes damage?

**Comment by Zunwolf1896** [comment]
Correct. the person still gets set on fire and takes the fire damage.

**Comment by Zunwolf1896** [comment]
We are using Minecraft 1.20.1
blossom-lib-2.5.4+1.20
blossom-pvp-2.0.1+1.20
Fabric-api-0.92.2+1.20.1

**Comment by CodedSakura** [comment]
alright, thank you for the information
